src/environments/array.js

```diff
@@ -786,7 +786,31 @@ defineEnvironment({
         const symNode = checkSymbolNodeType(args[0]);
         const colalign: AnyParseNode[] =
             symNode ? [args[0]] : assertNodeType(args[0], "ordgroup").body;
-        const cols = colalign.map(function(nde) {
+        let myColAlign = [];
+        if (colalign[0].text === "*") {
+            const repeatCol = assertNodeType(colalign[1], "ordgroup");
+            let repeatString = "";
+            for (let i = 0; i < repeatCol.body.length ; i++) {
+                const iRepeat = assertNodeType(repeatCol.body[i], "textord");
+                repeatString += iRepeat.text ;
+            }
+            const nRepeat = parseInt(repeatString);
```

Comment:
Should use `Number` (or `parseInt(..., 10)`) to force decimal parsing.

I see that there's similar code (which uses `Number`) when computing `numMaths`. Could this be factored out into a helper function? (with the comment above too) Converting text to numbers like this is quite useful... In fact, I see a totally different implementation of it (which I maybe wrote?) in `macros.js` under `defineMacro("\\char"`. 😮